ovn-tester/ovn_utils.py

```diff
@@ -35,15 +119,26 @@ def add_port(self, port, bridge, internal=True, ifaceid=None):
     def del_port(self, port):
         self.run(f'del-port {port.name}')
 
+    def create_vm_port_addr_str(self, name, addr, plen, gw):
+        cmd = f'ip -n {name} addr add {addr}/{plen} dev {name} ; ' \
+              f'ip -n {name} route add default via {gw} ; '
```

Comment:
No need for whitespace before ';' (applies to the line above too).

Reply:
@dceara ack, I will fix it